### ISSUE TITLE
Some network updates are not available in the library

### ISSUE THREAD

**Issue by GoogleCodeExporter** [NityaNandPandey/Linkedin-SignIn#19]
```
Some network update types (e.g. recommendations given) are not available
through the API.

Both the domain model and the api needs to be updated to accomodate all
available network update types.
```

Original issue reported on code.google.com by `nabeelmukhtar` on 26 Dec 2009 at 6:14

**Comment by GoogleCodeExporter** [comment]
```
Added a few new updates types in linkedin-j-0.1-SNAPSHOT-2009-12-30. Job update 
is
still remaining.
```

Original comment by `nabeelmukhtar` on 30 Dec 2009 at 12:14
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Have a look at http://developer.linkedin.com/docs/DOC-1131. It has been updated.
```

Original comment by `nabeelmukhtar` on 6 Jan 2010 at 4:59

**Comment by GoogleCodeExporter** [comment]
```
This issue has been fixed in the build linkedin-j-0.1-SNAPSHOT-2010-01-07.zip. 
All
update types have been implemented as mentioned in the doc.
```

Original comment by `nabeelmukhtar` on 7 Jan 2010 at 7:19
- Changed state: **Fixed**
